cpu/nrf5x_common/include/gpio_ll_arch.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2015 Jan Wagner <mail@jwagner.eu>
+ *               2015-2016 Freie Universität Berlin
+ *               2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_nrf5x_common
+ * @ingroup     drivers_periph_gpio_ll
+ * @{
+ *
+ * @file
+ * @brief       CPU specific part of the GPIO NG API
```

Comment:
```suggestion
 * @brief       CPU specific part of the GPIO Low Level API
```
? Here an in the other header files.

Reply:
> ? Or something like that. Here an in the other header files.